packages/common-settings-connector/src/errors.ts

```diff
@@ -0,0 +1,35 @@
+/**
+ * Errors used in Common Settings Connector
+ * @class CommonSettingsError
+ * @extends Error
+ */
+export class CommonSettingsError extends Error {
+    constructor(message: string) {
+        super(message)
+        this.name = 'CommonSettingsError'
+    }
+}
+
+/**
+ * Error thrown when AMQP URL is not provided
+ * @class UrlNotProvidedError
+ * @extends CommonSettingsError
+ */
+export class UrlNotProvidedError extends CommonSettingsError {
```

Comment:
With the previous comment, https://github.com/linagora/ToM-server/pull/262/files#r2324899301, it can be renamed to something more generic than CommonSettingsError.